### PR TITLE
cobo-cli 0.0.4 (new formula)

### PR DESCRIPTION
**Pull request by wqvv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/cobo-cli.rb [hunk 1]
```diff
@@ -0,0 +1,169 @@
+class CoboCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Cobo Command-line Tool"
+  homepage "https://github.com/CoboGlobal/cobo-cli/blob/master/README.md"
+  url "https://github.com/CoboGlobal/cobo-cli/archive/refs/tags/v0.0.4.tar.gz"
```
Comment (chenrui333) [inline Formula/c/cobo-cli.rb:6]: can you do a pypi sdist for this?

File: Formula/c/cobo-cli.rb [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  sha256 "06903a3c056e38e8898c82601f8abb6bd53633b9de1ee86c8c41cb7e2d03e2ea"
+  license "MIT"
+
+  depends_on "libsodium"
+  depends_on "libyaml"
+  depends_on "python@3.9"
```
Comment (chenrui333) [inline Formula/c/cobo-cli.rb:12]: can it be built with py3.13?

Comment (wqvv) [reply Formula/c/cobo-cli.rb:12]: yes can . so i need update this to `depends_on "python@3.13"` ?

File: Formula/c/cobo-cli.rb (hunk 1 above)
Comment (wqvv) [reply Formula/c/cobo-cli.rb:6]: ok

File: Formula/c/cobo-cli.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/c/cobo-cli.rb:12]: yeah, indeed

File: Formula/c/cobo-cli.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/c/cobo-cli.rb:6]: thanks!

File: Formula/c/cobo-cli.rb (hunk 2 above)
Comment (wqvv) [reply Formula/c/cobo-cli.rb]: done

File: Formula/c/cobo-cli.rb (hunk 1 above)
Comment (wqvv) [reply Formula/c/cobo-cli.rb]: done

**Review by chenrui333 — APPROVED** [review]
looks good, can you run `brew update-python-resources cobo-cli` to refresh the dependencies? Thanks!

**Comment by wqvv** [comment]
> looks good, can you run `brew update-python-resources cobo-cli` to refresh the dependencies? Thanks!

i have already run `brew update-python-resources cobo-cli`  ,but there is no change about cobo-cli.rb

**Comment by wqvv** [comment]
> > looks good, can you run `brew update-python-resources cobo-cli` to refresh the dependencies? Thanks!
> 
> i have already run `brew update-python-resources cobo-cli` ,but there is no change about cobo-cli.rb

when this pr can be merged , thanks.

File: Formula/c/cobo-cli.rb [hunk 3]
```diff
@@ -0,0 +1,169 @@
+class CoboCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Cobo Command-line Tool"
+  homepage "https://github.com/CoboGlobal/cobo-cli/blob/master/README.md"
```
Comment (chenrui333) [inline Formula/c/cobo-cli.rb:5]:
```suggestion
  homepage "https://github.com/CoboGlobal/cobo-cli"
```

**Pull request closed without merging**